builtin/plugins/dposv3/storage.go

```diff
@@ -479,7 +479,7 @@ func saveState(ctx contract.Context, state *State) error {
 	return ctx.Set(stateKey, state)
 }
 
-func loadState(ctx contract.StaticContext) (*State, error) {
+func LoadState(ctx contract.StaticContext) (*State, error) {
```

Comment:
Why did this change?

Reply:
BeginBlock needs to access this function so that we can pass `state.Params.DowntimePeriod` to `UpdaeDowntime` and don't need to load state every function call